### ISSUE TITLE
How to fine tune the model without conversion to Hugging Face

### ISSUE THREAD

**Issue by xuchaozj** [facebookresearch/llama#797]
Stanford lab(https://github.com/tatsu-lab/stanford_alpaca) and meta research lab (https://github.com/facebookresearch/llama-recipes) provide the sample to fine tune the model requiring concersion to HF, is there a way to fine tune the model without the conversion?
Thanks

**Comment by LLaMA-Builder** [comment]
+1

**Comment by richardarcher1** [comment]
+1. We have been having significant issues despite plenty of GPU access. Even getting the logits from one forward pass is causing issues. 
If anyone has a clean way to split the model across multiple GPUs, I would love to hear (I've tried manually sending ~10 layers to each of 3 GPUs, but it has led to obscure error messages)

**Comment by BingyuanZhang** [comment]
+1
